src/components/Header/components/UpperNav.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { chakra, useMultiStyleConfig } from "@chakra-ui/react";
+
+import Button from "../../Button/Button";
+import Icon from "../../Icons/Icon";
+import Link from "../../Link/Link";
+import List from "../../List/List";
+
+const HeaderUpperNav = chakra(({ logInOpen, setLogInOpen }) => {
+  const styles = useMultiStyleConfig("HeaderUpperNav", {});
+  return (
+    <List
```

Comment:
Navigations should be wrapped in a `nav` element. When you do so, add `aria-label="Header top links"`. This is the aria label on the current header top nav.